### DISCUSSION TITLE
Open the description section on the first load

### DISCUSSION THREAD

**Discussion by Amorim33** [scalar/scalar#1953] [Ideas]
## Description

[Screencast from 2024-06-05 15-59-44.webm](https://github.com/scalar/scalar/assets/42624869/03ba257d-38fb-48f9-b6d5-fcba3b7707c5)

On the first load, the first Open API tag is open.

However, personaly speaking, when a description section is added with a H2 (##), the app is probably expected to either initiate with this section opened or initiate without any section opened.

**Comment by Amorim33** [comment]
If it makes sense I will be pleased to work on it at the weekend 😊

**Reply by hanspagel** [reply]
This makes so much sense! I’ve created an issue: https://github.com/scalar/scalar/issues/1989
